up42/order.py

```diff
@@ -200,42 +208,32 @@ def track_status(self, report_time: int = 120) -> str:
         Returns:
             str: The final order status.
         """
-
-        def substatus_messages(substatus: str) -> str:
-            substatus_user_messages = {
-                "FEASIBILITY_WAITING_UPLOAD": "Wait for feasibility.",
-                "FEASIBILITY_WAITING_RESPONSE": "Feasibility is ready.",
-                "QUOTATION_WAITING_UPLOAD": "Wait for quotation.",
-                "QUOTATION_WAITING_RESPONSE": "Quotation is ready",
-                "QUOTATION_ACCEPTED": "In progress.",
-            }
-
-            if substatus in substatus_user_messages:
-                message = substatus_user_messages[substatus]
-                return f"{substatus}, {message}"
-            return f"{substatus}"
-
+        substatus_user_messages = {
```

Comment:
I think this is not needed at all. But I did not want to change the current logs until we discuss.

Reply:
Agree

Reply:
I removed this.